python/sglang/srt/server.py

```diff
@@ -428,7 +428,11 @@ def launch_server(
     t.start()
 
     try:
+        from uvicorn.config import LOGGING_CONFIG
         # Listen for HTTP requests
+        LOGGING_CONFIG["formatters"]["default"]["fmt"] = "%(asctime)s [%(name)s] %(levelprefix)s %(message)s"
```

Comment:
does this show the date as well?

Reply:
Yes.
https://github.com/sgl-project/sglang/issues/1623#issuecomment-2413166385
In the above link, there is a comparison bewteen old and new log messages.

Reply:
is it possible to include the timezone info?